hudi-flink/src/test/java/org/apache/hudi/operator/CopyOnWriteTest.java

```diff
@@ -58,13 +57,13 @@
 /**
  * Test cases for StreamingSinkFunction.
  */
-public class StreamWriteFunctionTest {
+public class CopyOnWriteTest {
```

Comment:
Let us start with `Test` and give it a more valuable name?

Reply:
Rename to `TestWriteCopyOnWrite`.